spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
I was happy with Glenn changing it to add the ":" - if you want just the destination name, call getDestinationName(). In general I am not a fan of relying on what toString() does. If I really wanted a way to reconstruct identical DSL I'd have a method called toDSLString() across all the nodes.